pkg/storage/tsdb/ref_cache.go

```diff
@@ -0,0 +1,157 @@
+package tsdb
+
+import (
+	"sync"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/cortexproject/cortex/pkg/util"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/labels"
+)
+
+const (
+	// DefaultRefCacheTTL is the default RefCache purge TTL. We use a reasonable
+	// value that should cover most use cases. The cache would be ineffective if
+	// the scrape interval of a series is greater than this TTL.
+	DefaultRefCacheTTL = 5 * time.Minute
+
+	refCacheStripes = 128
```

Comment:
`refCacheStripes` --> `numRefCacheStripes`

Down in L29, it looks like an array of `refCacheStripe`.